starters/gatsby-wordpress-starter/lib/PostsPagination.js

```diff
@@ -0,0 +1,24 @@
+import { gql } from '@pantheon-systems/wordpress-kit'
+import { paginationClient } from './PaginationClient'
+
+export async function paginationPostsQuery() {
+	const query = gql`
+		query paginationPostsQuery {
+			posts(first: 50) {
+				edges {
+					node {
+						id
+						title
+						excerpt
+					}
+				}
+			}
+		}
+	`
+
+	const {
+		posts: { edges },
+	} = await paginationClient.request(query)
+
+	return edges.map(({ node }) => node)
+}
```

Comment:
Can we rename this file to camel case or skewer case please. I don't quite recall which we landed on but it should be in our code styles doc.